flake.nix

```diff
@@ -305,7 +305,7 @@
 
             nativeBuildInputs = with pkgs; [
               pkg-config
-              moreutils
+              (pkgs.writeShellScriptBin "ts" "exec ${pkgs.moreutils}/bin/ts \"$@\"")
```

Comment:
Could you make this a variable somewhere so it's not repeated thrice?

Reply:
Sure we can.